src/main/java/uk/gov/companieshouse/service/CompanyOfficerService.java

```diff
@@ -2,7 +2,6 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import uk.gov.companieshouse.api.model.officers.OfficerApi;
 import uk.gov.companieshouse.client.CompanyOfficersClient;
 import uk.gov.companieshouse.model.dto.companyOfficers.CompanyOfficer;
```

Comment:
I believe the companyOfficers and companyProfile packages also need to be converted to lowercase.

Reply:
I believe that is fixed in Sapphire's PR